srcpkgs/hub/template

```diff
@@ -1,31 +1,38 @@
 # Template file for 'hub'
 pkgname=hub
-version=2.7.1
+version=2.8.4
 revision=1
 build_style=go
 go_import_path=github.com/github/hub
-hostmakedepends="git ruby-ronn"
+hostmakedepends="git"
 short_desc="Command line tool for Github"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="MIT"
 homepage="https://hub.github.com/"
 changelog="https://github.com/github/hub/releases"
 distfiles="https://github.com/github/hub/archive/v${version}.tar.gz"
-checksum=83fac8b47b2f56da1e7366688df4bd0eeb5fa20bde5d3de39792ea1cc1d15cd4
+checksum=0aa1342ac5701dc9b7e787ad69640ede06fc84cbe88fb63440b81aca4d4b6273
 
 if [ "$CROSS_BUILD" ]; then
 	hostmakedepends+=" hub"
 fi
 
 post_install() {
-	# Don't use bundled ronn
-	mkdir bin
-	ln -s /usr/bin/ronn bin/ronn
-
 	if [ "$CROSS_BUILD" ]; then
 		vsed -e 's|bin/hub|hub|g' -i Makefile
 	fi
 
+	# Required for building md2roff for the host
+	case "$XBPS_MACHINE" in
+		aarch64*) export GOARCH=arm64;;
+		armv6*) export GOARCH=arm; export GOARM=6;;
+		armv7*) export GOARCH=arm; export GOARM=7;;
+		i686*) export GOARCH=386;;
+		x86_64*) export GOARCH=amd64;;
+		ppc64le*) export GOARCH=ppc64le;;
+		ppc64*) export GOARCH=ppc64;;
+	esac
```

Comment:
It'd be prettier if we did something like this in a build_helper or added it to the go build-style (maybe via (XBPS_)HOST_GOARCH?